README.md

```diff
@@ -1,2 +1,120 @@
-# speakleash-forum-tools
-A collection of tools used for gathering text data from discussion forums
+<h1 align="center">
+<img src="https://raw.githubusercontent.com/speakleash/speakleash/main/branding/logo/speakleash_logo.png" width="300">
+</h1><br>
+
+<p align="center">
+    <a href=""><img src="https://img.shields.io/badge/version-0.0.2-brightgreen"></a>
+    <a href="https://pypi.org/project/speakleash"><img src="https://img.shields.io/badge/python-_>=_3.6-blue"></a>
+    <a href="https://speakleash.org/"><img src="https://img.shields.io/badge/organisation-Speakleash-orange"></a>
+</p>
+
+# SpeakLeash Forum Tools
+
+## Overview
+
+SpeakLeash Forum Tools is a comprehensive toolkit designed for crawling, scraping, and managing data from various online forums. The package integrates multiple components such as configuration management, crawling, scraping, and archiving functionalities to streamline the process of extracting and organizing forum data.
+
+## Features
+
+- **Forum Crawling**: Efficiently crawl forums using sitemaps or manual crawling techniques.
+- **Data Scraping**: Extract text data from forum pages with support for pagination and different forum engines.
+- **Forum Engines Supports**: ['invision', 'phpbb', 'ipboard', 'xenforo', 'other'] - ‘other’ option is designed for more difficult forums, where appropriate parameters such as pagination or HTML paths with posts must be defined. 
+- **Data Archiving**: Manage and merge scraped data into well-organized archives.
+- **Manifest Management**: Create and handle SpeakLeash manifest files for datasets, including metadata and statistics.
+- **Documentation**: Each module and class is carefully documented to make it easy to understand and use.
+- **Detailed Logs**: Extensive logging capabilities to track the progress and status of crawling and scraping operations, including debugging and error information.
+
+## Installation
+
+Currently, this toolkit is available as a repository on GitHub. To use it, you need to clone the repository:
+
+```bash
+git clone https://github.com/speakleash/speakleash-forum-tools.git
+cd speakleash-forum-tools
+```
+
+For testing create new virtual environment using:
+```bash
+python -m venv .venv
+```
+
+After that activate virtual environment:
```

Comment:
I'd add a coma for better readability:
`After that, activate virtual environment:`